seaborn/tests/test_utils.py

```diff
@@ -368,59 +364,61 @@ def test_locator_to_legend_entries():
         assert str_levels == ['1e-07', '1e-05', '1e-03', '1e-01', '10']
 
 
-if LooseVersion(pd.__version__) >= "0.15":
-
-    def check_load_dataset(name):
-        ds = load_dataset(name, cache=False)
-        assert(isinstance(ds, pd.DataFrame))
-
-    def check_load_cached_dataset(name):
-        # Test the cacheing using a temporary file.
-        # With Python 3.2+, we could use the tempfile.TemporaryDirectory()
-        # context manager instead of this try...finally statement
-        tmpdir = tempfile.mkdtemp()
-        try:
-            # download and cache
-            ds = load_dataset(name, cache=True, data_home=tmpdir)
-
-            # use cached version
-            ds2 = load_dataset(name, cache=True, data_home=tmpdir)
-            pdt.assert_frame_equal(ds, ds2)
-
-        finally:
-            shutil.rmtree(tmpdir)
-
-    @_network(url="https://github.com/mwaskom/seaborn-data")
-    def test_get_dataset_names():
-        if not BeautifulSoup:
-            raise nose.SkipTest("No BeautifulSoup available for parsing html")
-        names = get_dataset_names()
-        assert(len(names) > 0)
-        assert(u"titanic" in names)
-
-    @_network(url="https://github.com/mwaskom/seaborn-data")
-    def test_load_datasets():
-        if not BeautifulSoup:
-            raise nose.SkipTest("No BeautifulSoup available for parsing html")
-
-        # Heavy test to verify that we can load all available datasets
-        for name in get_dataset_names():
-            # unfortunately @network somehow obscures this generator so it
-            # does not get in effect, so we need to call explicitly
-            # yield check_load_dataset, name
-            check_load_dataset(name)
-
-    @_network(url="https://github.com/mwaskom/seaborn-data")
-    def test_load_cached_datasets():
-        if not BeautifulSoup:
-            raise nose.SkipTest("No BeautifulSoup available for parsing html")
-
-        # Heavy test to verify that we can load all available datasets
-        for name in get_dataset_names():
-            # unfortunately @network somehow obscures this generator so it
-            # does not get in effect, so we need to call explicitly
-            # yield check_load_dataset, name
-            check_load_cached_dataset(name)
+def check_load_dataset(name):
+    ds = load_dataset(name, cache=False)
+    assert(isinstance(ds, pd.DataFrame))
+
+
+def check_load_cached_dataset(name):
+    # Test the cacheing using a temporary file.
+    # With Python 3.2+, we could use the tempfile.TemporaryDirectory()
```

Comment:
Implementing this would fit well with these changes.

Reply:
Added this as well, thanks!